resources/js/Components/Table/CharacterTable/CharacterTable.tsx

```diff
@@ -122,7 +123,7 @@ const CharacterTable = ({
         setSelected(newSelected)
     }
 
-    const isSelected = (name: string) => selected.indexOf(name) !== -1
+    const isSelected = (id: number) => selected.includes(id)
```

Comment:
please move this since this is not re-usable

Reply:
I thought I removed this in my last commit lol! Will remove it for sure this time.